From bbeef7f559bd9c6b1aad11bcd65e56428f290bd8 Mon Sep 17 00:00:00 2001
From: Michal Marek <mmarek@suse.cz>
Date: Wed, 26 Feb 2014 13:48:55 +0100
Subject: [PATCH 1/6] modprobe: Recognize --allow-unsupported-modules on
 commandline

The option does not do anything yet, but it does not return error
either.

References: fate#316971
Patch-mainline: never
---
 tools/modprobe.c | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tools/modprobe.c b/tools/modprobe.c
index a9e2331..3be2989 100644
--- a/tools/modprobe.c
+++ b/tools/modprobe.c
@@ -85,6 +85,8 @@ static const struct option cmdopts[] = {
 	{"dirname", required_argument, 0, 'd'},
 	{"set-version", required_argument, 0, 'S'},
 
+	{"allow-unsupported-modules", no_argument, 0, 128},
+
 	{"syslog", no_argument, 0, 's'},
 	{"quiet", no_argument, 0, 'q'},
 	{"verbose", no_argument, 0, 'v'},
@@ -843,6 +845,9 @@ static int do_modprobe(int argc, char **orig_argv)
 		case 'S':
 			kversion = optarg;
 			break;
+		case 128:
+			/* --allow-unsupported-modules does nothing for now */
+			break;
 		case 's':
 			env_modprobe_options_append("-s");
 			use_syslog = 1;
-- 
2.20.1

